apps/server/src/classes/data-provider/DataProvider.ts

```diff
@@ -86,7 +86,7 @@ function getCustomFields(): Readonly<CustomFields> {
 }
 
 async function setRundown(newData: OntimeRundown): ReadonlyPromise<OntimeRundown> {
-  db.data.rundown = [...newData];
+  db.data.rundown = newData;
```

Comment:
@cpvalente 
do we need to make a copy here?
we did not make a full copy befor?
and we don't do anything to this object